bugsnag-android-core/src/main/java/com/bugsnag/android/DeliveryDelegate.java

```diff
@@ -24,9 +24,9 @@
         this.breadcrumbState = breadcrumbState;
     }
 
-    void deliver(@NonNull Event event) {
```

Comment:
Making `Notifier` a member variable would avoid some of the method signature changes - I think this makes sense as it's more similar to the logger, config and breadcrumb state objects that you're setting in the constructor.

Reply:
Addressed similar comments by changing these to member variables.